cache/engine/client/client_cluster_test.go

```diff
@@ -0,0 +1,464 @@
+//go:build all || functional
+// +build all functional
+
+package client_test
+
+//                                                                         __
+// .-----.-----.______.-----.----.-----.--.--.--.--.______.----.---.-.----|  |--.-----.
+// |  _  |  _  |______|  _   |   _|  _  |_   _|  |  |______|  __|  _  |  __|     |  -__|
```

Comment:
this fixes the ascii art:

```suggestion
// |  _  |  _  |______|  _  |   _|  _  |_   _|  |  |______|  __|  _  |  __|     |  -__|
```

Reply:
done!